packages/atomic/src/components/search/atomic-generated-answer/placeholders.tsx

```diff
@@ -0,0 +1,60 @@
+import {h, Fragment, FunctionalComponent} from '@stencil/core';
+
+export const GeneratedAnswerPlaceholder: FunctionalComponent = () => {
+  return (
+    <Fragment>
+      <GeneratedAnswerHeaderPlaceholder />
+      <GeneratedAnswerParagraphPlaceholder />
+      <GeneratedAnswerCitationsPlaceholder />
+    </Fragment>
+  );
+};
+
+const GeneratedAnswerHeaderPlaceholder: FunctionalComponent = () => {
+  const classesAnim = 'block bg-neutral rounded h-4 animate-pulse';
+  const classesAnimHeaderActions = `${classesAnim} w-12`;
+  return (
+    <div class="flex items-center mb-8">
+      <div class={`${classesAnim} w-48`}></div>
+      <div class="flex gap-2 h-9 items-center ml-auto">
+        <div class={classesAnimHeaderActions}></div>
+        <div class={classesAnimHeaderActions}></div>
+        <div class={classesAnimHeaderActions}></div>
+      </div>
+    </div>
+  );
+};
+
+const GeneratedAnswerParagraphPlaceholder: FunctionalComponent = () => {
+  const classesAnim = 'block bg-neutral rounded h-4 animate-pulse mb-4';
+  return (
+    <div>
+      <div class={`${classesAnim} w-full`}></div>
+      <div class={`${classesAnim} w-3/4`}></div>
+      <div class={`${classesAnim} w-3/4`}></div>
+      <div class={`${classesAnim} w-full`}></div>
+      <div class={`${classesAnim} w-1/3`}></div>
+      <div class={`${classesAnim} w-full`}></div>
+      <div class={`${classesAnim} w-1/3`}></div>
+      <div class={`${classesAnim} w-full`}></div>
+      <div class={`${classesAnim} w-3/4`}></div>
+    </div>
```

Comment:
The animation for the facet values placeholders use random values instead of being hardcoded. Should we do the same here ? https://github.com/coveo/ui-kit/blob/master/packages/atomic/src/components/common/facets/facet-placeholder/facet-placeholder.tsx

Reply:
Ehhh, not a bad idea, but I'm not sure it would be ... "better" ? I'm not sure, it's kind of subjective I guess :D

Reply:
@alexprudhomme, scope-out this suggestion in your takeover. (If we're to do it I'd like @johnnyCoveo to review the result, but we don't have time to spare for that, tho @nathanlb if you think @alexprudhomme suggestion is nice, please feel free to pick it up ;) )